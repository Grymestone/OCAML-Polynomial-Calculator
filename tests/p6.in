(2a^3 + 7a^4 + 1a^2)^5
