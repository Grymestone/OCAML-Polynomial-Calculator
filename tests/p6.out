((((2*(a)^3)+(7*(a)^4))+(1*(a)^2)))^5
24010x^19 + 16807x^20 + 25725x^18 + 17640x^17 + 9870x^16 + 4092x^15 + 1410x^14 + 360x^13 + 75x^12 + 10x^11 + 1x^10